wrangler.toml

```diff
@@ -3,6 +3,8 @@ name = "houstonbot"
 main = "./src/index.ts"
 compatibility_date = "2023-11-08"
 
+node_compat = true
+
```

Comment:
Why do we need this?
We need to understand the underlaying issue and fix it, instead of using `node_compat`. It is not recommended to use it. It is kinda deprecated, without being officially deprecated.

Reply:
I believe I added this when we still had "discord.js" in out package.json. This can be removed now